60.md

```diff
@@ -50,6 +51,7 @@ Tags:
 * `description` - Optional human-readable description of the wallet.
 * `balance` - Optional best-effort balance of the wallet that can serve as a placeholder while an accurate balance is computed from fetching all unspent proofs.
 * `privkey` - Private key used to unlock P2PK ecash. MUST be stored encrypted in the `.content` field. **This is a different private key exclusively used for the wallet, not associated in any way to the user's nostr private key** -- This is only used when receiving funds from others, described in NIP-61.
+* `ncryptsec` - Provide an option to use either the above `privkey` or `ncryptsec`. This approach can help prevent the user's wallet `privkey` from being exposed in case the main nsec is leaked.
```

Comment:
```suggestion
* `ncryptsec` - Allows the user to encrypt the `privkey` so that input from the user is required to spend from this wallet. When the `ncryptsec` tag is used, no `privkey` should be available.
* `p2pk` - Public key of the key in the `ncryptsec` -- this tag should be provided when using encrypting the private key via `ncryptsef` so that apps can know which wallet contains the private key of the NIP-61 nutzap.
```